### ISSUE TITLE
Bash-language-server and explainshell container

### ISSUE THREAD

**Issue by pryorda** [mads-hartmann/bash-language-server#56]
Hello Everyone,

So I'm having some fun trying to get the bash language server with the explainshell integration working. 

I have the explainshell container running and my settings look like:
```
{
    "files.autoSave": "afterDelay",
    "editor.tabSize": 2,
    "files.exclude": {
        "**/.git": true,
        "**/.svn": true,
        "**/.hg": true,
        "**/CVS": true,
        "**/.DS_Store": true,
        "**/secrets": true
    },
    "ruby.lint": {
        "rubocop": true,
        "ruby": true, //Runs ruby -wc
    },
    "python.pythonPath": "/usr/local/opt/python@2/bin/python2.7",
    "python.linting.pylintEnabled": false,
    "python.linting.flake8Enabled": true,
    "python.linting.enabled": true,
    "python.linting.pylintPath": "/usr/local/bin/pylint",
    "bashIde.explainshellEndpoint": "http://localhost:5000"
}
``` 

If I browse to localhost:5000 in my browser everything works 100%. The container access logs only show the browser requests and I do not see bash-language-server trying to communicate at all.

Thoughts?

**Comment by skovhus** [comment]
Hm... which Docker image did you use?

**Comment by pryorda** [comment]
```
docker run --rm --name bash-explainshell -p 5000:5000 chrismwendt/codeintel-bash-with-explainshell
```

**Comment by pryorda** [comment]
VS says the bashIde.explainshellEnpoint is not a valid configuration option

**Comment by pryorda** [comment]
Looks like its just not getting the flag documentation.

**Comment by skovhus** [comment]
Wondering if you misspelled `Endpoint`?

**Comment by pryorda** [comment]
My configuration is . for . above.

**Comment by maximbaz** [comment]
It took me a while as well to figure why explainshell is not working, turns out the latest published version `1.5.1` simply doesn't include the explainshell functionality yet.

@mads-hartmann would you please consider publishing a new version? Ideally after merging #58 😛
